main.nf

```diff
@@ -22,57 +22,60 @@ def helpMessage() {
+      --max_insert_size                  Maximum insert size of mapped reads to consider. Default: None
+      --saveInteractionBAM               Save BAM file with interaction tags (dangling-end, self-circle, etc.). Default: False
+
+      --dnase                            Run DNase Hi-C mode. All options related to restriction fragments are not considered. Default: False
+
+      --min_cis_dist                     Minimum intra-chromosomal distance to consider. Default: None
+      --rm_singleton                     Remove singleton reads. Default: true
+      --rm_multi                         Remove multi-mapped reads. Default: true
+      --rm_dup                           Remove duplicates. Default: true
+
+      --bin_size                         Bin size for contact maps (comma separated). Default: '1000000,500000'
+      --ice_max_iter                     Maximum number of iteration for ICE normalization. Default: 100
+      --ice_filter_low_count_perc        Percentage of low counts columns/rows to filter before ICE normalization. Default: 0.02
+      --ice_filter_high_count_perc       Percentage of high counts columns/rows to filter before ICE normalization. Default: 0
+      --ice_eps                          Convergence criteria for ICE normalization. Default: 0.1
 
     Other options:
-      --splitFastq			    Size of read chuncks to use to speed up the workflow
-      --outdir				    The output directory where the results will be saved
-      --email                       	    Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits
-      -name                         	    Name for the pipeline run. If not specified, Nextflow will automatically generate a random mnemonic.
+      --splitFastq                       Size of read chuncks to use to speed up the workflow. Default: None
+      --outdir                           The output directory where the results will be saved. Default: './results'
+      --email                            Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits. Default: None
+      -name                              Name for the pipeline run. If not specified, Nextflow will automatically generate a random mnemonic. Default: None
 
     Step options:
-      --skip_maps                           Skip generation of contact maps. Useful for capture-C
-      --skip_ice			    Skip ICE normalization
-      --skip_cool			    Skip generation of cooler files
-      --skip_multiQC			    Skip MultiQC
+
+      --skip_maps                        Skip generation of contact maps. Useful for capture-C. Default: False
+      --skip_ice                         Skip ICE normalization. Default: False
+      --skip_cool                        Skip generation of cool files. Default: False
+      --skip_multiQC                     Skip MultiQC. Default: False
```

Comment:
ok. I did it for the skip* options, but It would indeed be great if we could fixe a rule for nf-core pipelines as most of my other options are in the form `boolean_value`

Reply:
Thanks! It's one of those unwritten rules that we've discussed and implemented to various degrees in some of the main pipelines but haven't had the time to officially document :sweat_smile: You could change the other parameters too? If it helps, if things do change then I'll have to update numerous pipelines! I've also updated the template today to change the last remaining parameter to this format...
https://github.com/nf-core/tools/pull/425/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR13

Reply:
I'll leave it up to you. :+1:

Reply:
ok. I open a new issue, and will do it in the next release if it's fine with you